tests/Dommel.Bulk.IntegrationTests/SqLiteBulkInsertTests.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using Dapper;
+using Dommel.Bulk.Tests.Common;
+using Microsoft.Data.Sqlite;
+using Xunit;
+
+namespace Dommel.Bulk.IntegrationTests;
+
+public class SqLiteBulkInsertTests : BulkInsertTestsBase<SqLiteAllTypesEntity>
+{
+    public SqLiteBulkInsertTests()
+    {
+        SqlMapper.AddTypeHandler(new GuidHandler());
```

Comment:
I think this behavior should be documented. You can also add a setting that fixes the problem.